package.json

```diff
@@ -42,8 +42,12 @@
   ],
   "devDependencies": {
     "@lhci/cli": "^0.10.0",
-    "beachball": "^2.31.0",
+    "beachball": "2.31.6",
```

Comment:
With the other package updates, I started getting errors due to the beachball/storybook patched package versions not matching what the patches wanted. With these changes, I no longer got those errors when I did an npm install. But similar to my comment on the package-lock, there's probably a better change to make to handle this.

Reply:
(If we wait for #1361 to go in, we won't need any changes in this file, see [here](https://github.com/ni/nimble/commit/715655ccd2b626de1bfea94038a87cec2ca60ca8), so can ignore these changes for now)

Reply:
No longer any package.json changes.